std_msgs/Header header
float64 min_x
float64 min_y
float64 z_value
float64 grid_size
uint32 grid_width_num
TerrainGrid[] grids